frontend/components/bc/BcText.vue

```diff
@@ -23,7 +24,9 @@ withDefaults(
 
 <style scoped lang="scss">
 @use '~/assets/css/breakpoints' as *;
-
+.reset {
+  margin-block: unset;
```

Comment:
necessary as `h2` tags get `margin-block` from `user-agent-styles`...should probably be removed on a `global level` though